### DISCUSSION TITLE
target间存在构建顺序，但是不需要链接怎么写呢？

### DISCUSSION THREAD

**Discussion by heheda123123** [xmake-io/xmake#5841] [General]
有个target a是共享库，有个target b是可执行程序
现在希望在构建b的时候把a复制到另一个位置做处理，但是b并不需要链接a。该怎么写呢？
用了add_deps可以设置构建顺序，但是会把a链接到b上

**Comment by star-hengxing** [comment]
`add_deps("a", {inherit = false})`
